src/Reporting/src/App.Metrics.Formatting.StatsD/Internal/StatsDFormatterConstants.cs

```diff
@@ -0,0 +1,20 @@
+﻿// <copyright file="StatsDFormatterConstants.cs" company="App Metrics Contributors">
+// Copyright (c) App Metrics Contributors. All rights reserved.
+// </copyright>
+
+using System;
+
+namespace App.Metrics.Formatting.StatsD.Internal
+{
+    public class StatsDFormatterConstants
+    {
+        public static string ItemTagName = "item";
+        public static string SampleRateTagName = "sampleRate";
+        public static string TimestampTagName = "timestamp";
+
+        public static class GraphiteDefaults
```

Comment:
Small thing, class name from copy/paste?

Reply:
Oh, right, i'll fix that in a jiffy.